### ISSUE TITLE
Modules already initialized bug

### ISSUE THREAD

**Issue by rodrigoargumedo** [jneen/rouge#290]
I am experiencing a bug with `1.9.1` with some console output in the Travis log in one of my [repos](https://travis-ci.org/rodrigoargumedo/personal-website-rails):

```ruby
/home/travis/build/rodrigoargumedo/personal-website-rails/vendor/bundle/ruby/2.1.0/gems/rouge-1.9.1/lib/rouge/lexers/shell.rb:20: warning: already initialized constant Rouge::Lexers::Shell::KEYWORDS
/home/travis/build/rodrigoargumedo/personal-website-rails/vendor/bundle/ruby/2.1.0/gems/rouge-1.9.1/lib/rouge/lexers/shell.rb:20: warning: previous definition of KEYWORDS was here
/home/travis/build/rodrigoargumedo/personal-website-rails/vendor/bundle/ruby/2.1.0/gems/rouge-1.9.1/lib/rouge/lexers/shell.rb:25: warning: already initialized constant Rouge::Lexers::Shell::BUILTINS
/home/travis/build/rodrigoargumedo/personal-website-rails/vendor/bundle/ruby/2.1.0/gems/rouge-1.9.1/lib/rouge/lexers/shell.rb:25: warning: previous definition of BUILTINS was here
```

I did confirm that this bug is already on development and testing environment with Rails. It seems a bit strange to get this particular warnings about already initialized `Rouge::` constant modules.

**Comment by canon4ever** [comment]
After i run the `bundle update`, i got the information too.
I don't know how should i solve it.

**Comment by rodrigoargumedo** [comment]
This fixes in 471294a
